app/core/store/modules/interactives.js

```diff
@@ -0,0 +1,85 @@
+import { getInteractive, getSession } from 'ozaria/site/api/interactive'
+
+export default {
+  namespaced: true,
+
+  state: {
+    loading: {
+      interactive: false,
+      session: false
+    },
+
+    interactive: undefined,
+    interactiveSession: undefined
+  },
+
+  mutations: {
+    toggleInteractiveLoading (state) {
+      state.loading.interactive = !state.loading.interactive
+    },
+
+    toggleInteractiveSessionLoading (state) {
+      state.loading.session = !state.loading.session
+    },
+
+    addInteractive (state, interactive) {
+      state.interactive = interactive
+    },
+
+    addInteractiveSession (state, interactiveSession) {
+      state.interactiveSession = interactiveSession
+    }
+  },
+
+  getters: {
+    currentInteractiveDataLoading (state) {
+      return state.loading.interactive || state.loading.session
+    },
+
+    currentInteractive (state) {
+      return state.interactive
+    },
+
+    currentInteractiveSession (state) {
+      return state.interactiveSession
+    }
+  },
+
+  actions: {
+    async loadInteractive ({ commit }, interactiveIdOrSlug) {
+      commit('toggleInteractiveLoading')
+
+      try {
+        const interactive = await getInteractive(interactiveIdOrSlug)
+        if (!interactive) {
+          throw new Error('Invalid interactive received')
+        }
+
+        commit('addInteractive', interactive)
+      } catch (e) {
+        // TODO handle
```

Comment:
Replace with new i18n comment  tag

Reply:
Is it related to updating after we have a consistent error handling strategy? If yes, recommend using the same comment as this: https://github.com/codecombat/codecombat/commit/e067b291e37cf304e405d2505a00a031847c1cd9#diff-586d801665ed7d9b31d90eb82124927aR33